runtime/config.go

```diff
@@ -34,7 +34,10 @@ type Config struct {
 	// ResourceOwnerChangeCallbackEnabled configures if the resource owner change callback is enabled
 	ResourceOwnerChangeHandlerEnabled bool
 	// CoverageReportingEnabled configures if coverage reporting is enabled
+	// Deprecated: Use the `CoverageReport` field instead
 	CoverageReportingEnabled bool
```

Comment:
Given that the field is not used anymore, just remove it
```suggestion
```

Reply:
Good point :+1:  I have removed it entirely in this commit https://github.com/onflow/cadence/pull/2416/commits/cc5644b671468ee11075b1a81cd1d7797a895b2a